apollo-ios/Sources/ApolloAPI/SchemaTypes/InputObject.swift

```diff
@@ -44,3 +44,5 @@ public struct InputDict: GraphQLOperationVariableValue, Hashable {
   }
 
 }
+
+public protocol OneOfInputObject: InputObject { }
```

Comment:
Went ahead and added this protocol stub to help differentiate the regular and oneOf input objects in the generated code, and also in case we need to add customizations or anything to oneOf objects specifically in the future.

Reply:
I think this is a good idea.